src/Mailgun/Model/Suppression/Unsubscribe/IndexResponse.php

```diff
@@ -21,11 +21,25 @@
     use PaginationResponse;
 
     /**
+     * Array to store a list of Unsubscribe items from
+     * index response
+     * 
+     * @see Mailgun/Model/Suppression/Unsubscribe/Unsubscribe
      * @var Unsubscribe[]
      */
-    private $items;
+    private $items = [];
```

Comment:
IMHO we don't need this. Constructor already expects an array and the value will be set as an array.

Reply:
The advantage of initializing properties outside of the constructor is that someone reading your code will immediatly know its a default value.

Also for automatic analysis or help in IDE.

I don't think there is much of a difference when it comes to performance and it indeed helps to newbies from IDE or so

Reply:
AFAIK IDE's are using the annotation against the value, so `Unsubscribe[]` defines an array of `Unsubscribe` objects, and a developer will know too that we're dealing with an array.